src/assets/assets.md

```diff
@@ -22,6 +28,98 @@ owned by an account and can be sent to other users.
 * TOC
 {:toc}
 
+## Asset Model
+
+All assets have the following fields along with the additional fields that are
+specific to its category.
+
+**Required Fields**
```

Comment:
It feels a little strange to call these "required". On other pages in our APIs, this term sets out an expectation on our API consumer to provide those fields. However, this is instead a requirement on our API response.

Could we add words or use a different title here?

Maybe something like "Guaranteed response fields"?